addons/context_actions/XEH_PREP.hpp

```diff
@@ -6,6 +6,10 @@ PREP(canPasteVehicleAppearance);
 PREP(canRearmVehicles);
 PREP(canRefuelVehicles);
 PREP(canRepairVehicles);
+PREP(canSwitchWeapon);
+PREP(canSwitchWeaponBinocular);
+PREP(canSwitchWeaponHandgun);
+PREP(canSwitchWeaponPrimary);
```

Comment:
These functions can be combined since they are very similar. Probably also don't need a condition on the main action since it will not be shown if no children actions are active.

Reply:
[1d3a562](https://github.com/zen-mod/ZEN/pull/520/commits/1d3a562cbe834a480a57118547d541dc0614cb40)
great points!